modules/apps/frontend-data-set/frontend-data-set-views-web/src/main/resources/META-INF/resources/css/Filters.scss

```diff
@@ -0,0 +1,3 @@
+.filter-by-cell-renderers-dropdown-menu {
+	cursor: pointer;
+}
```

Comment:
Hmm, this is a bit of a smell here, typically we shouldn't need any custom css when using clay components

Reply:
Thanks for notice, I was using an Input for dropdown trigger. Changing it for a button as in Fields.tsx solves it, so we can get rid of the entire scss file